src/test/java/com/example/homework10/service/MusicServiceImplTest.java

```diff
@@ -58,7 +58,7 @@ public class MusicServiceImplTest {
 
     @Test
     public void ミュージックが作成されること() {
-        Music music = new Music(5, "明日への扉", "kiroro");
+        Music music = new Music(0, "明日への扉", "kiroro");
```

Comment:
Musicクラスに以下のようなコンストラクターが用意されていましたので、こっちのコンストラクタを使っちゃって良いかな？と思いました！
```
    public Music(String title, String singer) {
        this.title = title;
        this.singer = singer;
    }
```
```suggestion
        Music music = new Music("明日への扉", "kiroro");
```

Reply:
確かにidの情報が入らなければこちらのコンストラクターでも問題ないですもんね🤔
使用するコンストラクターを変更しました。
[89bc87d](https://github.com/yuu3415/homework10/pull/6/commits/89bc87d8de3d0e266a4ab1b11c6c91dd3b2824bf)